code/widgets/org.eclipse.scout.widgets.client/src/main/java/org/eclipse/scout/widgets/client/ui/forms/CalendarComponentForm.java

```diff
@@ -130,6 +171,26 @@ protected String getConfiguredLabel() {
       }
     }
 
+    @Order(3500)
+    @ClassId("db5e6404-df17-4c97-92ec-5cd944d6cb4b")
+    public class CalendarField extends AbstractSmartField<CalendarResourceDo> {
+      @Override
+      protected String getConfiguredLabel() {
+        return TEXTS.get("Calendar");
```

Comment:
I have renamed the field, but would not rename the display text. Mostly because the label "Kalenderauswahl" would not match then. Is this fine with you?

Reply:
Ok, I agree!